R/input-slider.R

```diff
@@ -296,6 +298,37 @@ findStepSize <- function(min, max, step) {
   }
 }
 
+# Throw a warning if ever `value` is not in the [`min`, `max`] range
+validate_slider_value <- function(min, max, value, fun) {
+  if (is.null(min)   || is_na(min) ||
+      is.null(max)   || is_na(max) ||
+      is.null(value) || is_na(value))
+  {
+    stop(call. = FALSE,
+      sprintf("In %s(): `min`, `max`, and `value` cannot be NULL or NA.", fun)
+    )
+  }
```

Comment:
If we're inspecting this.... 

Won't catch `numeric(0)`

```suggestion
  if (
    length(min) == 0 ||
    length(max) == 0) ||
    length(value) == 0 ||
    is.null(min)   || is_na(min) ||
    is.null(max)   || is_na(max) ||
    is.null(value) || is_na(value))
  {
    stop(call. = FALSE,
      sprintf("In %s(): `min`, `max`, and `value` cannot be NULL, NA, or empty.", fun)
    )
  }
```